renovate.json

```
@@ -1,7 +1,6 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": ["config:best-practices"],
```

```suggestion
  "extends": ["config:best-practices", "helpers:pinGitHubActionDigestsToSemver"],
```

This basically makes renovate's version comments for github action digest always be for an exact version, like this:
```
 uses: 8398a7/action-slack@1750b5085f3ec60384090fb7c52965ef822e869e # v3.18.0
```

That way, any update PRs it creates for actions will include an updated comment, and also a link to a changlog in the summary PR.